spec/plugin_spec.rb

```diff
@@ -38,6 +38,14 @@ def expect_user_tl0_post_to_be_queued(post, state = "confirmed_ham")
     Jobs::CheckAkismetPost.clear
   end
 
+  it "destroys reviewable when post is deleted by user destroyer class" do
+    post = Fabricate(:post)
+    reviewable = ReviewableAkismetPost.new(target: post)
+    UserDestroyer.new(admin).destroy(post.user, delete_posts: true)
+
+    expect { reviewable.reload }.to raise_error(ActiveRecord::RecordNotFound)
+  end
```

Comment:
Shouldn't we have this spec in `user_destroyer_spec.rb` ?

Reply:
Thanks, I moved it to `user_destroyer_spec.rb ` file.